mlc_llm/core.py

```diff
@@ -483,7 +483,7 @@ def build(mod_deploy: tvm.IRModule, args: argparse.Namespace) -> None:
                 )
             mod_deploy = dl.ApplyDefaultSchedule(  # pylint: disable=not-callable
                 dl.gpu.Matmul(),
-                dl.gpu.GEMV(),
+                # dl.gpu.GEMV(),
```

Comment:
Commented out since the GeMV rule fails on AutoGPTQ-quantizated decode-GeMV.

**Please do not merge.**

Reply:
Good to merge as long as https://github.com/mlc-ai/relax/pull/284 is merged.